.gitignore

```diff
@@ -21,7 +21,7 @@ node_modules
 !.vscode/*.code-snippets
 
 # Logs
-logs
+#logs
```

Comment:
Yeah I will remove before merge, just wanted confirmation from others first

Reply:
I can't think of why that would be a problem, but I would prefer the `logger` name over `logs` anyway (but that's just personal preference). In case you would like to stop ignoring logs folders, check other ignore files too (.prettierignore, .nxignore).

Reply:
Sure, I will rename it

Reply:
@matejkriz @Nodonisko Renamed to logger both in common and native folders https://github.com/trezor/trezor-suite/pull/7359/commits/3f3c1222a4e5885134d7bea5ced565b2bf0c7d47